dev-docs/source/Testing/MuonInterface/MuonTesting.rst

```diff
@@ -1,224 +1,190 @@
+	
+- Back in the workspaces toolbox, click the workspace group for the fitted,
+  data (Name similar to *EMU51341; Pair Asym; long; MA+ ...; Fitted; FlatBackground,DynamicKuboToyabe*)
+- Ctrl + Click all the Workspace 2D data, then right click **Plot** > **Spectrum ...**
+- Choose Plot type **Individual** and click **Plot All**
+- You should get something like the following plot:
+
+.. figure:: ../../images/MuonAnalysisTests/MATestingIDF.png
+	:alt: MATestingIDF.png
+
+--------------------------------
+
+.. _superconducting_copper_test:
 
 Superconducting Copper Test
 ---------------------------
 
-*Preparation*
-
--  Files ``EMU00020882.nxs`` – ``EMU00020900.nxs``
--  Make sure the location of these files is included in your search
-   directories
-
 **Time required 5 - 10 minutes**
 
---------------
-
-1. Make sure that the ``.nxs`` files are in one of the directories on
-   Mantid’s search path.
-
--  Open Muon Analysis
--  Choose ``Machine``: ``EMU``
--  In ``Load Run`` enter ``20882``
--  Click the ``>`` tab next to ``Load Run`` a few times, this should
-   produce new plots each time
--  Under ``Settings`` tab, ensure the ``Enable Multiple Fitting`` is
-   un-checked
--  After loading ``20884`` switch to the ``Data Analysis`` tab
--  Right click under ``Property`` and select ``Add Function``
--  Choose ``Muon`` > ``ExpDecayMuon``
--  Click ``Fit`` > ``Fit``
--  Click ``Fit`` > ``Sequential Fit``
--  Beside ``Runs`` enter ``20889-20900``
--  Click ``start``
--  Close the ``Sequential Fits`` window
--  Switch to the ``Results Table`` tab
--  In the ``Fitting Results`` table check the ``Sequential`` radio
-   button
--  In the ``Values`` table check ``Run Number`` and ``Field_Danfysik``
--  Ensure that all runs are checked in the ``Fitting Results`` table
--  Click ``Create Table``
--  In the main Mantid GUI select the ``ResultsTable`` in the
-   ``Workspaces`` tab, this is the one you just created
--  Right click and select ``Convert to MatrixWorkspace``
--  In the resulting dialog select ``Column X`` as ``Field_Danfysik`` and
-   ``Column Y`` as ``lambda``
--  Click ``Run``
--  Right click the new workspace created in the ``Workspaces`` tab
--  Select ``Plot Spectrum``
--  You should get something like the following plot:
-
-.. figure:: ../../images/Cu-fitting.png
+- Open **Muon Analysis** (*Interfaces* > *Muon* > *Muon Analysis*)
+- Change *Instrument* to **EMU**, found in the *Home* tab
+- Load run ``20882``
+- Using the **>** button, step through a couple files (Plot should
+  update each time)
+- Go to the **Fitting** tab
+	- Right click the empty table area; Select **Add Function**
+	- Add **ExpDecayMuon** (*Muon* > *MuonGeneric* > 
+	  *ExpDecMuon*)
+- Go to the **Sequential Fitting** tab
+	- Click **Sequentially Fit All**
+- Go to the **Results** tab
+	- In the **Log Values** table, check **run_number** and **Field_Danfysik**
+	- Click **Output Results**
+- In the main Workbench window, in the **Algorithms Toolbox**, search for the 
```

Comment:
Instead you can open the results table, select the `field_danfysik` column and the lambda column while pressing control. Both should be highlighted. Right click the lambda column and select plot-> line